lib/canvas_link_migrator/link_parser.rb

```diff
@@ -19,7 +19,6 @@
 
 require "nokogiri"
 require "digest"
-require "byebug"
```

Comment:
Maybe we should add that to the gemspec as a dev_dependency :)

Reply:
Lol it's already there actually, I just never checked and threw it in willy-nilly.